vars/RedHat.yml

```diff
@@ -1,2 +1,4 @@
 ---
 php_xdebug_module_path: /usr/lib64/php/modules
+php_extension_conf_path: /etc/php.d
+__php_webserver_daemon: "httpd"
```

Comment:
I think you forgot to copy&paste the code to use the default.
As it is now this will fail on RedHat and make the php_webserver_daemon un-overridable on debian.